assets/airports/eidw.json

```diff
@@ -883,442 +1076,766 @@
       "end":    [["N53d26m13.16", "W6d15m43.12", "217ft"], ["N53d25m11.66", "W6d14m58.54", "202ft"]],
       "length":      2.072,
       "ils":         [true, true]
+    },
+    {
```

Comment:
For the time being, I think having the SIDs on the current runways are fine. At the moment, we don't have documentation for players and I don't think this would work particularly well